### DISCUSSION TITLE
Help ensto chago wallbox configuration

### DISCUSSION THREAD

**Discussion by maui80cc** [lbbrhzn/ocpp#321] [General]
Hi guys, I'm new to both home assistant and this integration. I have an ensto chago wallbox that I would like to do work with this integration since it does not have its own proprietary app and I would like to manage it remotely. I installed the integration, but I can not make it communicate with the server, I would need a hand to configure it, I am also available to make a remote connection through TeamViewer. The wallbox is compatible with all occp protocols

**Comment by maui80cc** [comment]
![Uploading IMG_20211111_114632.jpg…]()

**Reply by lbbrhzn** [reply]
your image upload doesn't seem to work. You should be able  to copy and paste into the discussion thread.

**Reply by lbbrhzn** [reply]
Some hints to get things working:

In your charger configuration (webpage or app), your should select you want to use OCCP v1.6, over JSON. The url for occp should be set to the IP address of your homeassistant server, and the port to 9000 (or the port you have configured when setting up the integration).

i.e. url = 192.168.1.100:9000

Some chargers need a / at the end:

url = 192.168.1.100:9000/

Some charger need the ws:// prefix:

url = ws://192.168.1.100:9000

You home assistant server should preferably run on hassos on linux, people have reported issues with windows.
If you run on a Linux flavor different from hassos you may need to open up the port (9000) in your firewall.
You can check that the server port is open and that the integration on the server is running successfully by using a websocket client, like Websocket Weasel https://addons.mozilla.org/en-US/firefox/addon/websocket-weasel/

**Reply by maui80cc** [reply]
> your image upload doesn't seem to work. You should be able to copy and past into the discussion thread.

I can't copy the image from my phone

**Reply by maui80cc** [reply]
I put in the address of the configuration page of the wallbox the address ws://homeassistant.local:9000, instead I have to put the ip address of ha

**Reply by lbbrhzn** [reply]
Its better to use the IP address, not every router will convert homeassistant.local to the correct ip.

**Reply by maui80cc** [reply]
I work on the Debian Linux e ha core

**Reply by maui80cc** [reply]
Tonight I post the photos of the configuration page and advise me how to set it. Thank you

**Reply by lbbrhzn** [reply]
That should work fine. With netstat-a you should be able to see if there is a service listening at port :9000.
If there is, you should check if the port is open from the outside, it may be blocked by a firewall.
Just browse to the homeassistant.local:9000 (or the ip address)
Should return something like:

```
Failed to open a WebSocket connection: invalid Connection header: keep-alive.
You cannot access a WebSocket server directly with a browser. You need a WebSocket client.
```

**Reply by maui80cc** [reply]
Ok, now I see the entities, but I don't see the most important one and that is the one that allows me to change the power of the station

**Reply by maui80cc** [reply]
![Screenshot_20220121_222542_io homeassistant companion android](https://user-images.githubusercontent.com/77131192/150602543-b85b2558-0139-4739-9109-6dc4cad758e1.jpg)

**Reply by maui80cc** [reply]
Do you have a Solution?

**Reply by lbbrhzn** [reply]
@maui80cc We have a known issue with the max charging slider remaining inadvertently disabled. I have created a workaround and released a beta version v0.4.9. Can you try it?

**Reply by maui80cc** [reply]
Yes i can try

**Reply by ernst77** [reply]
> Yes i can try

Hi, did you manage to connect everything? I am currently trying to connect same ensto wallbox charger, would like to know if there is any limitations?
